config/aws.docker.yml

```diff
@@ -0,0 +1,22 @@
+# config/aws.yml
```

Comment:
What is this file? 
It looks like our old portal config files.

Reply:
I think this is just vestigial and doesn't read well in the diff.  The `aws.docker.yml` file is the renamed `aws.yml.docker` file with the dev/test secrets removed and instead all environments use the ENV vars.

Reply:
The old `aws.yml.docker` file looked like this:

```

# config/aws.yml
base: &base
  access_key_id: ''       # your access key id
  secret_access_key: ''   # your secret access key
  s3:
    bucket: ''              # your bucket name
    region: 'us-east-1'     # region prefix of your bucket url, eg. "s3-eu-west-1"
    expires: '3600'         # number of seconds from now when signed urls expire
  transcoder:
    pipeline_id: ''         # transcoder pipeline id (created in aws console)

development:
  <<: *base

test:
  <<: *base

cucumber:
  <<: *base

production:
  access_key_id: <%= ENV['ACCESS_KEY_ID'] %>
  secret_access_key: <%= ENV['SECRET_ACCESS_KEY'] %>
  s3:
    bucket: <%= ENV['S3_BUCKET'] %>
    region: <%= ENV['S3_REGION'] %>
    expires: '3600'
  transcoder:
    pipeline_id: <%= ENV['PIPELINE_ID'] %>
```

Reply:
Oh, is this the config file that the AWS sdk is loading?
I'm just confused because I thought this app would just load ENV variables directly and not have any indirection through a config file.

Reply:
Yes, this is an intermediary file.  The newer AWS library uses this method to setup the global config that is passed to the other gems (in v3 each service is its own gem)

```
Aws.config.update({
  region: aws_config["s3"]["region"],
  credentials: Aws::Credentials.new(aws_config["access_key_id"], aws_config["secret_access_key"])
})
```

you can can just set the environment variables but I was just upgrading from the old version in which I don't believe you could.